march_joint_inertia_controller/CMakeLists.txt

```diff
@@ -0,0 +1,209 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(march_joint_inertia_controller)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
Please remove all comments from this file as they make it hard to read it. If anyone needs help on how to write a CMakeLists.txt they can look it up.